### PR TITLE
Makin website

### PR DESCRIPTION
**Pull request by AnaBristow**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1 +1,38 @@
-Coming soon
+<!doctype html>
+<html>
+  <head>
+      <title>Just Think Charity</title>
+      <link rel="stylesheet" type="text/css" href="style.css">
+      <link href="https://fonts.googleapis.com/css?family=Advent+Pro" rel="stylesheet">
+  </head>
+  <img class="logo" src="logo.jpg">
+  <body>
```
Comment (Dean177) [inline docs/index.html:9]: The indentation is quite inconsistent, tags which have the same parent should have the same level of indentation

**Review by Dean177 — APPROVED** [review]

File: docs/index.html [hunk 2]
```diff
@@ -1 +1,38 @@
...
+          THERE IS NO MONEY INVOLVED, this is a FREE non profitable charity. Simply use any of the contact information
+            we've provided and we will set up a time and date to come and pick up your donations! If we are unable to answer
+            the phone please leave a message and we will get back to you as soon as possible.
+        Please remember to share this page as much as possible to help spread the word.</p>
+    </section>
+  <footer>
+      <ul>
+          <li>Phone: 07784416732</li>
+          <li>Email: justthinkcharity@hotmail.com</li>
+          <li>Facebook: https://www.facebook.com/JustThinkCharity/</li>
```
Comment (Dean177) [inline docs/index.html:34]: We should make this an actual link`<a>`

File: docs/index.html (hunk 1 above)
Comment (AnaBristow) [reply docs/index.html:9]: I                        agree.
          I'll try harder             next        time.

File: docs/index.html (hunk 2 above)
Comment (AnaBristow) [reply docs/index.html:34]: yes i know
i was going to.....

Comment (AnaBristow) [reply docs/index.html:34]: YOU'RE NOT MY SUPERVISOR

File: docs/index.html [hunk 3]
```diff
@@ -1 +1,38 @@
...
+          to the next step and create the Just Think charity.
+          THERE IS NO MONEY INVOLVED, this is a FREE non profitable charity. Simply use any of the contact information
+            we've provided and we will set up a time and date to come and pick up your donations! If we are unable to answer
+            the phone please leave a message and we will get back to you as soon as possible.
+        Please remember to share this page as much as possible to help spread the word.</p>
+    </section>
+  <footer>
+      <ul>
+          <li>Phone: 07784416732</li>
+          <li>Email: justthinkcharity@hotmail.com</li>
```
Comment (Dean177) [inline docs/index.html:33]: https://developer.mozilla.org/en/docs/Web/HTML/Element/a#Creating_an_email_link

File: docs/index.html (hunk 1 above)
Comment (Dean177) [reply docs/index.html:9]: You can do this with `Ctrl+Shift+Alt+L`

File: docs/style.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+h1, h2 {
+    font-color: gray;
+    font-family: "Advent Pro", sans-serif;
+}
+
+body {
+   display: flex;
+    flex-direction: column;
+    align-items: center;
+}
```
Comment (Dean177) [inline docs/style.css:47]: Try `Ctrl+Shift+Alt+L` on this file too

File: docs/index.html [hunk 5]
```diff
@@ -1 +1,38 @@
...
+        From that night, with the extra interest from our friends and family, we promised to take our ideas and visions
+          to the next step and create the Just Think charity.
+          THERE IS NO MONEY INVOLVED, this is a FREE non profitable charity. Simply use any of the contact information
+            we've provided and we will set up a time and date to come and pick up your donations! If we are unable to answer
+            the phone please leave a message and we will get back to you as soon as possible.
+        Please remember to share this page as much as possible to help spread the word.</p>
+    </section>
+  <footer>
+      <ul>
+          <li>Phone: 07784416732</li>
```
Comment (Dean177) [inline docs/index.html:32]: https://developer.mozilla.org/en/docs/Web/HTML/Element/a#Creating_a_phone_link

**Pull request merged**